scripts/tests/test_validate_unwanted_patterns.py

```diff
@@ -10,7 +10,7 @@ class TestStringsWithWrongPlacedWhitespace:
         "data",
         [
             (
-                """
+                r"""
```

Comment:
Noticed that without having this as a raw string, the test will fail on Python 3.12.